sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -302,7 +303,30 @@ def test_lasso_cv_positive_constraint():
     assert min(clf_constrained.coef_) >= 0
 
 
-# FIXME: 'normalize' to be removed in 1.2
+def _scale_alpha(estimator, n_samples):
+    """"Rescale the parameter alpha from when the estimator is evoked with
+    normalize set to True to when it is evoked in a Pipeline with normalize set
+    to False and with a StandardScaler."""
+    if 'alpha' not in estimator.get_params():
+        return
+
+    model_name = estimator.__class__.__name__
+    if model_name in ['Lasso', 'LassoLars', 'MultiTaskLasso']:
```

Comment:
While this is not a problem for the tests, using code patterns that test such conditions on class names would silently cause hard to debug unexpected results if this function was called on subclasses those classes. I would rather avoid using such misleading code patterns by using `isinstance` instead:

```python
    if isinstance(estimator, (Lasso, LassoLars, MultiTaskLasso)):
        ...
```